main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
```

Comment:
Consider placing all input requests within a single function to create the desired output. A helper function could also be useful for repeated requestions.